.github/actions/setup-java/action.yml

```diff
@@ -1,18 +1,12 @@
-name: 'Setup Java'
-description: 'Adds Java and Maven preconfigured for Eclipse SET'
-inputs:
-  settings:
-    description: 'Path to Maven settings.xml to copy. Only copied if the file exists.'
-    required: false
-    default: '.github/settings.xml'
```

Comment:
Am I getting this correct:, this input is omitted as you've transitioned to an global and required maven settings.xml?

Reply:
Yes. I noticed there's a `GITHUB_ACTION_PATH` env variable available. That allows us to resolve the settings.xml in the setup-java action and removes the need for a per-repository settings file.